src/components/NoteDialogComponents/NoteDialogNoteWrapper.scss

```diff
@@ -1,10 +1,9 @@
 @import "constants/style";
 @import "components/Note/Note.scss";
-
-$note-dialog__scrollbar-height: calc(100vh - #{$header__height} - #{$padding--default} - #{$margin--default} - #{$margin--large});
+@import "routes/StackView/StackView.scss";
 
 .note-dialog__scrollbar {
-  max-height: calc(#{$note-dialog__scrollbar-height} - 206px);
+  max-height: calc(100% - #{$stack-view__header-height} - #{$stack-view__parent-note-height});
   width: calc(100vw - (2 * $column__border-width));
   overflow-y: auto;
   overflow-x: hidden;
```

Comment:
imo a bottom padding/margin would be great so that the last note does not stick at the bottom but has a bit space to breathe